go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/hashicorp/go-memdb v1.3.4
 	github.com/hashicorp/go-retryablehttp v0.7.5
 	github.com/hexops/gotextdiff v1.0.3
-	github.com/kong/deck v1.29.3-0.20231114000742-27bb40ad93d6
+	github.com/kong/deck v1.29.3-0.20231213152632-675a51c86460
```

Comment:
Shouldn't this library not rely on deck? 🤔

Reply:
Most of the integration tests unfortunately still rely on the deck cmd package to run the library components, rather than having the stripped-down CLI stand-in function introduced here for diff.